ts/@live-compositor/browser-render/rollup.config.mjs

```diff
@@ -7,24 +7,27 @@ export default [
     input: 'src/index.ts',
     output: {
       file: 'dist/index.js',
-      format: 'es'
+      format: 'es',
     },
     plugins: [
       typescript(),
       copy({
         targets: [
-          { src: 'src/generated', dest: 'dist' },
```

Comment:
Previously I was importing the wasm file directly in the code and the typings for the wasm file were not bundled automatically so I had to manually copy them. Since I've changed the way wasm is imported in the package, this copy is no longer needed.